lib/portus/deprecation_error.rb

```diff
@@ -0,0 +1,7 @@
+module Portus
+  class DeprecationError < StandardError
```

Comment:
Could you add some documentation ? Something simple like: "DeprecationError is the exception to be raised when certain functionality has been deprecated in the latest version and we want to provide migration instructions".